docs/feature-flags/sdks/android.md

```diff
@@ -67,16 +67,18 @@ EppoClient eppoClient = new EppoClient.Builder()
 
-| Field                     | Description                                                                                                              | Example                  |
-| ------------------------- | ------------------------------------------------------------------------------------------------------------------------ | ------------------------ |
-| `experiment` (string)     | An Eppo experiment key                                                                                                   | "recommendation_algo"    |
-| `subject` (string)        | An identifier of the subject or user assigned to the experiment variation                                                | UUID                     |
-| `variation` (string)      | The experiment variation the subject was assigned to                                                                     | "control"                |
-| `timestamp` (string)      | The time when the subject was assigned to the variation                                                                  | 2021-06-22T17:35:12.000Z |
-| `subjectAttributes` (map) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `{ "country": "US" }`    |
+| Field                     | Description                                                                                                              | Example                             |
+| ------------------------- | ------------------------------------------------------------------------------------------------------------------------ | ----------------------------------- |
+| `experiment` (string)     | An Eppo experiment key                                                                                                   | "recommendation-algo-allocation-17" |
+| `subject` (string)        | An identifier of the subject or user assigned to the experiment variation                                                | UUID                                |
+| `variation` (string)      | The experiment variation the subject was assigned to                                                                     | "control"                           |
+| `timestamp` (string)      | The time when the subject was assigned to the variation                                                                  | 2021-06-22T17:35:12.000Z            |
+| `subjectAttributes` (map) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `{ "country": "US" }`               |
+| `featureFlag` (string)    | An Eppo feature flag key                                                                                                 | "recommendation-algo"               |
+| `allocation` (string)     | An Eppo allocation key                                                                                                   | "allocation-17"                     |
```

Comment:
Seems like we keep duplicating this, might be worthwhile to create a separate page (at some point?)